content-src/lib/filters.js

```diff
@@ -0,0 +1,28 @@
+function createFilter(definition) {
+  return function(item) {
+    let result = true;
+    definition.forEach(test => {
+      if (!test(item)) {
+        result = false;
+      }
+    });
+    return result;
+  };
+}
+
+const URL_FILTERS = [
+  item => !!item.url,
+  item => item.url && item.url.length < 100,
+  item => /^https?:/i.test(item.url),
+  item => !(/^https?:\/\/localhost/i.test(item.url))
```

Comment:
Not sure if we need to add a `\/` at the end of localhost here. Something tells me that this would incorrectly catch on domains like "http://localhost-is-cool.com"

Also, do we want to try and filter out loopback URLs like "127.0.0.1" and "0.0.0.0"?